Dockerfile.1

```
@@ -0,0 +1,53 @@
+# ENV UV_PROJECT_ENVIRONMENT=/app 
+
+COPY lib-new lib-new
+# RUN uv --directory=lib-new export --format requirements.txt --no-editable --frozen | sed -e 's@^\.@# Elided: .@' > /tmp/lib-new-constraints.txt
+# RUN uv --directory=lib-new build -b /tmp/lib-new-constraints.txt -o /tmp/dist --wheel
+RUN cd lib-new && UV_PROJECT_ENVIRONMENT=/tmp/lib-new/venv uv sync --frozen --no-dev --compile-bytecode
+RUN find /tmp/lib-new/venv -type f
+
+COPY app-new app-new
+# RUN uv --directory=lib-new export --format requirements.txt --no-editable --frozen | sed -e 's@^\.@# Elided: .@' > /tmp/lib-new-constraints.txt
+# RUN uv --directory=lib-new build -b /tmp/lib-new-constraints.txt -o /tmp/dist --wheel
+RUN cd app-new && UV_PROJECT_ENVIRONMENT=/tmp/app-new/venv uv sync --frozen --no-dev --compile-bytecode
```

Hadolint (2.12.0)

[warning] Use WORKDIR to switch to a directory

(DL3003)